docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -12,7 +12,7 @@ This page introduces additional configurations for Intel® Processor Graphics (G
 
 ## Linux
 
-If you have installed OpenVINO Runtime via the installer, APT, or YUM, follow these steps to work with GPU:
+If you have installed OpenVINO Runtime from the archive file, follow these steps to work with GPU:
```

Comment:
![image](https://user-images.githubusercontent.com/77325899/183426652-a41a7b7b-0eed-4f23-9efc-f95953a069ce.png)
I do not have this install_dependencies folder in the prerelease Windows archive for 2022.1.1 @artanokhov